.travis.yml

```diff
@@ -1,2 +1,8 @@
 language: c
+before_install:
+- wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb && sudo dpkg -i erlang-solutions_1.0_all.deb
+- sudo apt-get update
+- sudo apt-get install elixir
+- sudo apt-get install erlang
+- yes | mix local.hex
```

Comment:
is it possible to push this down to `bin/setup`? actually i wonder if you can push this entire `before_install` block into `bin/setup`...